bundles/org.openhab.ui/web/build/build.js

```diff
@@ -10,12 +10,14 @@ const config = require('./webpack.config.js');
 
 const env = process.env.NODE_ENV || 'development';
 const target = process.env.TARGET || 'web';
+const maven = process.env.MAVEN || false;
+const outPath = maven ? '../target/classes/app' : 'www';
```

Comment:
Did you verify that this works without the `./`? I just assumed that `rm` required it since it was already there, but it doesn't actually make sense because as far as I know, the "`./` convention" only applies when you execute files. If you're on Linux/macOS, you can easily check that the folder is actually deleted.

Reply:
I have tested it on Linux, works fine without the `./` as expected.